src/main/java/com/aws/greengrass/mqttclient/MqttClient.java

```diff
@@ -83,10 +87,17 @@
     static final String MQTT_OPERATION_TIMEOUT_KEY = "operationTimeoutMs";
     static final int DEFAULT_MQTT_OPERATION_TIMEOUT = (int) Duration.ofSeconds(30).toMillis();
     static final String MQTT_MAX_IN_FLIGHT_PUBLISHES_KEY = "maxInFlightPublishes";
-    static final int DEFAULT_MAX_IN_FLIGHT_PUBLISHES = 1;
+    static final int DEFAULT_MAX_IN_FLIGHT_PUBLISHES = 5;
     public static final int MAX_SUBSCRIPTIONS_PER_CONNECTION = 50;
     public static final String CLIENT_ID_KEY = "clientId";
     public static final int EVENTLOOP_SHUTDOWN_TIMEOUT_SECONDS = 2;
+    static final int DEFAULT_PUBLISH_RETRIED_COUNTER = 100;
+    // http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718023
+    static final int MAXIMUM_SIZE_OF_MQTT_MESSAGE_SIZE = 268_435_455;
```

Comment:
https://docs.aws.amazon.com/general/latest/gr/iot-core.html#message-broker-limits

IoT max message size is 128kb

Reply:
I think the max message size limit is about to increase. @MikeDombo Or it is better to set this to the current limit 128 kB for now?

Reply:
I'm reticent to put in that limit given that IoT Core may change it. Need to talk offline.